### ISSUE TITLE
Automate the process of checking Ohara manager's code style

### ISSUE THREAD

**Issue by konekoya** [oharastream/ohara#242]
We now have an npm script -- `prepush` for running mutiple tasks before pushing to remote repo:

- Ohara manager unit tests and linting 
- Ohara manager client unit tests and linting
- Gradle license apply task
- Prettier format apply

This is all great. However, one downside to this is that this task/script needs to be run manually. So we plan to move some of the tasks as a build step (server-side already runs a handful tasks in this) in CI build. This probably will include:

- Linting
- License check

Basically, we want to automate as many tasks as possible. As this can reduce changes of a bug/regression from getting into our codebase.

**Comment by konekoya** [comment]
dupe of #839
